dark/assets/css/styles.css

```diff
@@ -1051,6 +1051,10 @@ html {
   z-index: -1;
 }
 
+.fa{

```

Comment:
Can you use bootstrap class instead of custom css @Rishav1707

Reply:
I will try once then I'll report to you.

Reply:
https://github.com/YoshithaRathnayake/NEW-YOSHITHA-WEBSITE/blob/bbb62296bd27270313436aa40de85fb2fb471136/dark/index.html#L242

Add px-1 inside the class tag @Rishav1707

Reply:
Sorry I am unable to use bootstrap class instead of custom CSS. @YoshithaRathnayake

Reply:
> https://github.com/YoshithaRathnayake/NEW-YOSHITHA-WEBSITE/blob/bbb62296bd27270313436aa40de85fb2fb471136/dark/index.html#L242
> 
> Add px-1 inside the class tag @Rishav1707

Here I already tought you no? @Rishav1707

Reply:
One min

Reply:
Just remove the custom css and add this px-1 to the class tag @Rishav1707 

> https://github.com/YoshithaRathnayake/NEW-YOSHITHA-WEBSITE/blob/bbb62296bd27270313436aa40de85fb2fb471136/dark/index.html#L242
> 
> Add px-1 inside the class tag @Rishav1707

Reply:
> One min

Sure

Reply:
Done

Reply:
In #156